pyinstrument/low_level/stat_profile.c

```diff
@@ -265,6 +269,30 @@ call_target(ProfilerState *pState, PyFrameObject *frame, int what, PyObject *arg
     Py_DECREF(args);
     return result;
 }
+#else
+static PyObject *
+call_target(ProfilerState *pState, PyFrameObject *frame, int what, PyObject *arg)
+{
+    PyObject *whatstr;
+    PyObject *result;
+
+    whatstr = whatstrings[what];
+    Py_INCREF(whatstr);
+
+    PyFrame_FastToLocals(frame);
+    Py_INCREF(frame);
```

Comment:
I don't think these increfs are needed any more, I believe they were there because the PyTuple_SET_ITEM steals a refcount.

Reply:
Yeah, I wondered about that. I'll remove those.